src/platform/Darwin/BUILD.gn

```diff
@@ -65,7 +65,6 @@ static_library("Darwin") {
     "DnssdImpl.h",
     "DnssdType.cpp",
     "InetPlatformConfig.h",
-    "LoggingImpl.cpp",
```

Comment:
Why this is removed from platform sources, but it not added to "logging" target like in other platforms?

Reply:
Fixed